sdk/management/README.md

```diff
@@ -0,0 +1,221 @@
+# Azure management client library for Java
+
+The Azure Management Library for Java is a higher-level, object-oriented API for *managing* Azure resources,
+that is optimized for ease of use, succinctness and consistency.
+
+- [API reference documentation][docs]
+- [Samples][samples]
+
+## Getting started
+
+### Prerequisites
+
+- [Java Development Kit (JDK)][jdk] with version 8 or above
+- [Azure Subscription][azure_subscription]
+
+### Include the package
+
+[//]: # ({x-version-update-start;com.azure:azure-management;current})
+```xml
+<dependency>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-management</artifactId>
+  <version>2.0.0</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+### Include optional packages
+
+[Azure Identity][azure_identity] provides Azure Active Directory token authentication support across the Azure SDK.
+
+[//]: # ({x-version-update-start;com.azure:azure-identity;current})
+```xml
+<dependency>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-identity</artifactId>
+  <version>1.1.0-beta.3</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+[Azure Core Netty HTTP client][azure_core_http_netty] is a plugin for [Azure Core][azure_core] HTTP client API.
```

Comment:
i think it's worth to a separate section to let user know they need choose from 2 http client by themselves, and reference to each of them

Reply:
We can optionally refer to https://github.com/Azure/azure-sdk-for-java/wiki/HTTP-clients

I've checked other libs when writing this, they generally only mention it https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/storage/azure-storage-common#default-http-client